static/index.html

```diff
@@ -304,7 +304,7 @@
                   >
                 </li>
                 <li class="nav-item">
-                  <a class="nav-link" href="/blog/FAQ" target="#">FAQ</a>
+                  <a class="nav-link" href="/blog/FAQ" target="https://10101.finance/blog/faq/">FAQ</a>
```

Comment:
I think this is wrong. 

The `target` attribute specifies where the linked document will open when the link is clicked. The default is the current window. I guess it should be either `_blank` or `_self`. The former one opens in a new window, the second one opens in the same

```suggestion
                  <a class="nav-link" href="/blog/FAQ" target="_blank">FAQ</a>
```


```suggestion
                  <a class="nav-link" href="/blog/FAQ" target="_self">FAQ</a>
```

However, from what I can tell, it works in production for me.